### PR TITLE
Add TiCDC open protocol decoder

### PR DESCRIPTION
**Pull request by sunxiaoguang**
Support decoding TiCDC open protocol v1.

@humengyu2012 @marsishandsome PTAL

### CODE REVIEW

**Review by marsishandsome — COMMENTED** [review]
better to 
- add `mvn clean compile -am -pl ticdc` in `.ci/build.sh`
- add `mvn clean test -am -pl ticdc` in `.ci/test.sh`

File: ticdc/src/main/java/io/tidb/bigdata/cdc/Misc.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.tidb.bigdata.cdc;
+
+import io.tidb.bigdata.cdc.json.JsonNode;
+import io.tidb.bigdata.cdc.json.JsonParser;
+import io.tidb.bigdata.cdc.json.jackson.JacksonParserFactory;
```
Comment (marsishandsome) [inline ticdc/src/main/java/io/tidb/bigdata/cdc/Misc.java:21]: Unused import statement

**Comment by sunxiaoguang** [comment]
> better to
> 
> * add `mvn clean compile -am -pl ticdc` in `.ci/build.sh`
> * add `mvn clean test -am -pl ticdc` in `.ci/test.sh`

done

**Review by marsishandsome — APPROVED** [review]
others LGTM

File: ticdc/src/main/java/io/tidb/bigdata/cdc/Misc.java (hunk 1 above)
Comment (sunxiaoguang) [reply ticdc/src/main/java/io/tidb/bigdata/cdc/Misc.java]: Oops, thanks for pointing out. It's fixed now.

**Comment by humengyu2012** [comment]
LGTM

**Pull request merged**